src/main/java/us/ihmc/handsros2/abilityHand/AbilityHandManager.java

```diff
@@ -144,13 +155,43 @@ public AbilityHandManager(AbilityHandInterface hand)
    {
       this.hand = hand;
       goalPositions = new float[] {30.0f, 30.0f, 30.0f, 30.0f, 30.0f, -30.0f};
-      goalVelocities = new float[] {0.0f, 0.0f, 0.0f, 0.0f, 0.0f, 0.0f};
+      goalVelocities = new float[] {30.0f, 30.0f, 30.0f, 30.0f, 30.0f, 30.0f};
+
+      fingerTrajectories = new TrapezoidalTrajectory1D[ACTUATOR_COUNT];
+      for (int i = 0; i < ACTUATOR_COUNT; i++)
+      {
+         fingerTrajectories[i] = new TrapezoidalTrajectory1D(goalPositions[i], Math.abs(goalVelocities[i]), DEFAULT_MAXIMUM_ACCELERATION);
+         fingerTrajectories[i].reset(hand.getActuatorPosition(i), hand.getActuatorVelocity(i));
+      }
    }
 
    /** {@inheritDoc} */
    @Override
    public void update()
    {
+      long nowNanos = System.nanoTime();
+
+      if (previousTimeNanos > 0L)
+      {
+         long deltaNanos = nowNanos - previousTimeNanos;
+         float dt = Math.min(0.1f, Math.max(deltaNanos * 1.0e-9f, 0.001f));
```

Comment:
Should we make this change?
```suggestion
         float dt = Math.min(0.1f, deltaNanos * 1.0e-9f);
```